MobiFlight/Config/BaseDevice.cs

```diff
@@ -23,5 +24,9 @@ public DeviceType Type
         {
             get { return _type; }
         }
+        public bool isMuxClient
```

Comment:
Please remove this from the base class.
You can use Reflection to find that out if really needed.

What are we using this check for?

Reply:
See the previous reply to line 12 of the same file.